docs/3_developers/1_requesting_infrastructure.md

```diff
@@ -0,0 +1,14 @@
+# Requesting Infrastructure
```

Comment:
@negz @jbw976 @prasek any ideas on other things that could be helpful in this section?

Reply:
What's the context of this section? Is it a preface before we jump into static and dynamic provisioning guides? Should it distinguish between the app dev and app operator personas?

Reply:
I was initially just trying to put something with the developer section, but if we move `Packaging an Application` to here I think we can delete this. Not sure what goes with `Application Operator` though?